DeAceroDLabs/marketplace-frontend
# fix: [DMP-505] /signup view is not visible when logged in.

## What is done

[[DMP-505] Navigation arrows on categories](https://deacero.atlassian.net/browse/DMP-505)

## How is done

- Added username variable in `Signup` so that this one will send the user to home page if it is already logged in, if not the signup page appears.

## How to test

1. Run the project via `npm start`
2. Before logging in click on `Registrarse`, the signup view should be shown.
3. Log in
4. write /signup directly in url.
5. You should see home page / .
